internals/secrethub/run.go

```diff
@@ -404,7 +404,7 @@ func NewEnv(r io.Reader, vars map[string]string) (EnvSource, error) {
 			return nil, templateError(envvar, err)
 		}
 
-		template, err := parser.Parse(envvar.value, envvar.lineNumber, envvar.columnNumber)
+		template, err := parser.Parse(envvar.value, envvar.lineNo, envvar.valColNo)
```

Comment:
What's the reason for changing `Number` to `No`?

Reply:
I didn't like the long name `columnNumberValue`. But `No` might be unclear.

I'll change it.